core/model_regression.py

```diff
@@ -2,9 +2,31 @@
 from sklearn.pipeline import make_pipeline
 from sklearn.preprocessing import PolynomialFeatures
 from sklearn.exceptions import ConvergenceWarning
+from sklearn.metrics import mean_absolute_error
 from sklearn.utils._testing import ignore_warnings
+from core.model_base import BaseModel
 
 
+class StaticRegression(BaseModel):
+
+    def __init__(self, value):
+        self._value = value
+
+    def fit(self, train_features, train_labels):
+        return
+
+    def predict(self, test_features):
+        return [self._value] * len(test_features)
+
+    def score(self, test_features, test_labels):
+        return mean_absolute_error(test_labels, self.predict(test_features))
+
+
+def train_static_regression(train_features, train_label, value=1e-8):
+    model = StaticRegression(value=value)
+    model.fit(train_features, train_label)
+    return model
```

Comment:
value is configurable, basically all accepted claims will be assigned the same value